src/components/test-android.js

```diff
@@ -78,21 +73,33 @@ class TestAndroid extends React.Component {
                     )} */}
 
                     {this.props.runTest && (
-                        <div className="data__top-margin">
+                        <div className={layoutStyles.data__topMargin}>
                             {this.props.dataConstants.map(test => (
                                 <div key={test.state}>
                                     <Grid container>
                                         <Grid item xs={10}>
-                                            <small className="subitem-title">
+                                            <small
+                                                className={
+                                                    itemStyles.subitemTitle
```

Comment:
Why does this classname use a different naming style than the others?

Reply:
this was resolved in other commit